notebooks/model_tutorial/0_baseline_CNN.ipynb

```diff
@@ -2,44 +2,68 @@
  "cells": [
```

Comment:
<p>".. look at the .."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sinzlab/sensorium/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>fixed</p>